guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -7,6 +7,11 @@ You must have a Red{nbsp}Hat subscription manifest containing a subscription all
 All subscription information is available in your Red Hat Customer Portal account.
 
 Before you can complete the tasks in this chapter, you must create a Red{nbsp}Hat subscription manifest in the Customer Portal.
+Manifests can only be created on the Customer Portal by {Project} users on a disconnected network.
+Manifests are created by {Project} users on a connected network using the {Team} Hybrid Cloud Console.
```

Comment:
This shouldn't use `{Team}` because in upstream it creates `Foreman community Hybrid Cloud Console`.